manifests/t/tinygo-org/tinygo/0.30.0/tinygo-org.tinygo.locale.en-US.yaml

```
@@ -0,0 +1,43 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: tinygo-org.tinygo
+PackageVersion: 0.30.0
+PackageLocale: en-US
+Publisher: The TinyGo Authors
+PublisherUrl: https://tinygo.org
+PublisherSupportUrl: https://github.com/tinygo-org/tinygo/issues
+PackageName: TinyGo
+PackageUrl: https://tinygo.org/getting-started/install/windows
+License: Apache-2.0
+LicenseUrl: https://github.com/tinygo-org/tinygo/blob/release/LICENSE
+Copyright: Copyright (c) The TinyGo Authors
+CopyrightUrl: https://tinygo.org
+ShortDescription: TinyGo
+Description: TinyGo brings the Go programming language to embedded systems and to the modern web by creating a new compiler based on LLVM.

```

```suggestion
ShortDescription: Go compiler for small places. Microcontrollers, WebAssembly (WASM/WASI), and command-line tools. Based on LLVM.
Description: |-
  TinyGo is a Go compiler intended for use in small places such as microcontrollers, WebAssembly (wasm/wasi), and command-line tools.

  It reuses libraries used by the Go language tools alongside LLVM to provide an alternative way to compile programs written in the Go programming language.
```

ShortDescription should be something descriptive that's shown to user in `winget show` command